benchmarks/benchmark_functions/default_settings.py

```diff
@@ -49,3 +54,49 @@ def Template(params_):
 		template = Template
 
 	return device, diff_method, interface, params, template, measurement
+
+
+def _vqe_defaults(hyperparams):
+	"""Uses hyperparameters or defaults to construct the components of the VQE circuit for the
+	hydrogen molecule with the sto-3g basis set.
```

Comment:
I wonder if we should include the basis set as a possible hyperparameter. @soranjh If you recall, when we were benchmarking these functions manually, we benefited from playing around with different basis sets...

I guess the issue is that we have to build the Hamiltonian for each one, right? :thinking:

Reply:
Yes, exactly. If we make basis set a hyperparameter, then we should call an external program to compute the HF matrix elements with that basis set. We decided against including such external calls in our benchmarks.